Formula/biosig.rb

```diff
@@ -31,22 +32,52 @@ class Biosig < Formula
     sha256 "75df4a79b8d3d785942cbfd125ce45de49c3e7fa2cd19adb70caf8c4e30e13f0"
   end
 
+  patch :DATA
+
   def install
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}"
-    system "make"
+    system "./configure", "CC=gcc-10", "CXX=g++-10", "--disable-debug",
+           "--disable-dependency-tracking",
+           "--disable-silent-rules",
+           "--prefix=#{prefix}"
+    system "make", "CC=gcc-10", "CXX=g++-10"
     system "make", "install"
   end
 
   test do
     assert_match "usage: save2gdf [OPTIONS] SOURCE DEST", shell_output("#{bin}/save2gdf -h").strip
     assert_match "mV\t4274\t0x10b2\t0.001\tV", shell_output("#{bin}/physicalunits mV").strip
     assert_match "biosig_fhir provides fhir binary template for biosignal data",
-      shell_output("#{bin}/biosig_fhir 2>&1").strip
+                 shell_output("#{bin}/biosig_fhir 2>&1").strip
     testpath.install resource("test")
     assert_match "NumberOfChannels", shell_output("#{bin}/save2gdf -json TEST_44x86_e1.GDF").strip
     assert_match "NumberOfChannels", shell_output("#{bin}/biosig_fhir TEST_44x86_e1.GDF").strip
   end
 end
+
+__END__
+
+diff --git a/Makefile.in b/Makefile.in
+index 38726564..3531a1e7 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -93,14 +93,14 @@ first ::
+ 	make -C biosig4c++ mexbiosig
+ install ::
+ 	# mexbiosig
+-	#-@OCTAVE@ --no-gui --eval "pkg install -global biosig4c++/mex/mexbiosig-@PACKAGE_VERSION@.src.tar.gz"
++	-@OCTAVE@ --no-gui --eval "pkg install -global biosig4c++/mex/mexbiosig-@PACKAGE_VERSION@.src.tar.gz"
```

Comment:
Why is this patch needed, has it been reported upstream?

Reply:
This is an attempt to add the octave binding within the same package - however it does not seem to work.

Reply:
I'm upstream, so yes, I'm aware of this. The official release installs the octave-binding of biosig into a subdirectory of octave. homebrew does not seem to allow this. This patch prevents "make install" to fail.